lib/rdoc/generator/template/rails/_panel.rhtml

```diff
@@ -1,6 +1,9 @@
+<input type="checkbox" id="hamburger" class="panel_checkbox">
+<label class="panel_mobile_button" for="hamburger"><span></span> Menu</label>
```

Comment:
This is pretty cool! I've heard of checkboxes being used for this, but never seen it IRL!

How does it work from a `aria-expanded` standpoint?

Reply:
While looking at the implementation of the menu of https://stimulus.hotwire.dev/ I saw they used checkboxes this way.
I have no idea about the `aria-expanded` standpoint.

Reply:
If basecamp does it, I'd totally trust them to have made the right choice.

I'll quickly deploy this locally & give it a try using VoiceOver on my mobile to confirm we're a-ok :)